modules/effects/arrow-effects-kotlinx-coroutines-instances/src/main/kotlin/arrow/effects/DeferredKInstances.kt

```diff
@@ -81,7 +81,7 @@ interface DeferredKBracketInstance : Bracket<ForDeferredK, Throwable>, DeferredK
 @extension
 interface DeferredKMonadDeferInstance : MonadDefer<ForDeferredK>, DeferredKBracketInstance {
   override fun <A> defer(fa: () -> DeferredKOf<A>): DeferredK<A> =
-    DeferredK.defer(fa = fa)
+    DeferredK.defer(fa = { fa() })
```

Comment:
Unnecessary allocation. We can just pass `fa`, without the named parameter right?

Reply:
This was necessary because I changed signature of `defer` to `f: CoroutineScope.() -> DeferredK<A>` but this seems to not be necessary.

When reverting this back I solved the actual issue. I will create a seperate PR with the fix, since this is already merged and I don't want to do it in the `asyncF` PR. I will also add an explanation there.